src/api.rs

```diff
@@ -301,6 +301,7 @@ pub enum PredictionModesSetting {
   ComplexAll,
 }
 
+/// Contain all details of chroma sampling for different types like 4:2:0, 4:2:2, 4:4:4, 4:0:0
```

Comment:
how about just `different types of chroma sampling`?

just listing the types themselves in the documentation itself will only increase maintenance cost, so it's better to either exclude them or document them inline

Reply:
Documenting Inline will be better imo.